README.md

```diff
@@ -1,5 +1,36 @@
 The Spectral Sequences Project
 ==============================
 
-TODO: Readme here.
+This is a monorepo containing various projects:
 
+1. `ext`
+A general library to work with Ext over an Fp algebra. More generally, it
+allows us to work compute in the derived category of said algebra. The primary
+purpose is to compute the classical Adams E2 page by computing Ext over the
+Steenrod algebra.
+
+2. `web_ext`
+Web interfaces to `ext`. There are two subprojects at the moment:
+
+ - `rust_webserver`: A GUI to work with the Adams spectral sequence. Given a

```

Comment:
Do you think naming the webapps after their paths makes sense? E.g. this would be `sseq`? (I just think `rust_webserver` is not particularly suggestive of what the project is, since it's more webapp than webserver.)

Reply:
Yup, agreed.